app/(tabs)/(home)/index.tsx

```diff
@@ -122,6 +123,12 @@ export default function HomeScreen() {
         onPress={() => {
           setButtonState(ButtonStates.NotCycling);
           void stopAutomaticTracking();
+          router.push({
+            pathname: "../(touren)/stage",
+            params: {
+              stageId: activeStageId,
+            },
+          });
```

Comment:
```suggestion
          // @ts-ignore
          navigation.navigate("(touren)", {
            screen: "stage",
            params: { stageId: activeStageId },
            initial: false,
          });
```

Um den Bug zu fixen, wenn der Tour Screen noch nicht gemountet wurde. Expo Router's push unterstützt das inital: false nicht. Das bezweckt, dass der Screen nicht als erstes auf den Stack gepusht wird.

Du musst auch noch navigation hier platzieren und natürlich von expo-router importieren:
```js
export default function HomeScreen() {
  const navigation = useNavigation();
  const insets = useSafeAreaInsets();
```